src/utils/durableUtils.ts

```diff
@@ -229,48 +238,46 @@ export namespace netheriteUtils {
         return taskJson?.hubName;
     }
 
-    // Todo: Uncomment out in future PR
-    // export async function validateConnection(context: IActionContext, options?: Omit<IValidateConnectionOptions, 'suppressSkipForNow'>, projectPath?: string): Promise<void> {
-    //     projectPath ??= await getRootWorkspacePath();
-    //     if (!projectPath) {
-    //         throw new NoWorkspaceError();
-    //     }
-
-    //     const eventHubsConnection: string | undefined = await getLocalConnectionString(context, ConnectionKey.EventHub, projectPath);
-    //     const hasEventHubsConnection: boolean = !!eventHubsConnection && !localEventHubsEmulatorConnectionRegExp.test(eventHubsConnection);
+    // Supports validation on both 'debug' and 'deploy'
+    export async function validateConnection(context: IActionContext, projectPath: string, options?: IValidateConnectionOptions): Promise<void> {
+        const eventHubsConnection: string | undefined = await getLocalConnectionString(context, ConnectionKey.EventHub, projectPath);
```

Comment:
Yeah, the names on this are super weird, but I think it's technically correct this way.

So Event Hubs is considered the top level, and houses namespaces called Event Hubs Namespaces.

![image](https://user-images.githubusercontent.com/40250218/212415373-517016dd-bb21-4418-9da4-a7b3209e6d2c.png)

![image](https://user-images.githubusercontent.com/40250218/212415809-e5e2061b-6af1-4e9d-a377-2177fe8766a0.png)

Within Event Hubs Namespaces are Event Hubs, or Event Hub if singular.  So they're actually referencing different things, it's just super confusing because the only differentiation is an 's'....

Reply:
That probably means I should change `ConnectionKey.EventHub` to `ConnectionKey.EventHubs` though...  🫠

Reply:
So is the `eventHubsConnection` connecting to the entire namespace rather than a specific event hub?

Reply:
I believe so